src/controller.rs

```diff
@@ -63,13 +72,25 @@ impl FoxBox {
                http_port: Option<u16>,
                ws_port: Option<u16>) -> Self {
 
+        let hostname = hostname.map_or(DEFAULT_HOSTNAME.to_owned(), |name| {
+            format!("{}{}", name, DEFAULT_DOMAIN)
+        });
+
+        let local_crt = PathBuf::from(format!("certs/server/{}-server.crt.pem", hostname));
+        let local_key = PathBuf::from(format!("certs/server/{}-server.key.pem", hostname));
+
+        let remote_crt = PathBuf::from(format!("certs/server/{}-server.crt.pem", "test.knilxof.org"));
+        let remote_key = PathBuf::from(format!("certs/server/{}-server.key.pem", "test.knilxof.org"));
```

Comment:
Could we get this string from the `tunnel` command line option like we do with the box hostname?